blog/2021-12-06-itchysats-roadmap.md

```diff
@@ -0,0 +1,157 @@
+
+Once we have implemented these features we reach the milestone **Mainnet Stable** which completes a first stable product.
+After Mainnet Stable we will focus on making the solution **awesome**. The following chapters outline what we have planned.
+
+### Fly me to the moon
+
+In this stage we will focus on opening the market for multiple makers, and making CFD positions transferable.
+This involves being able to trade CFDs in limit orders and trading open positions in a decentral orderbook.
+
+### Reach for the stars
+
+In this stage we will focus on oracles: Partner with multiple oracle provicers to make the solution more
+
+
+### Fullblown P2P DEX
+
+This is the goal. Once we reach this milestone we have a full peer to peer Bitcoin DEX including a decentralized orderbook and multiple oracle. 
+You will be able to open a CFD on pretty much anything that has a value in Bitcoin by then.
+
+This does not mean that our journey ends here. 
+We have lots of ideas how to extend, scale up, improve what will exist then. 
+But once we reach this milestone we should have something pretty awesome.
+
+## Under the Hood: Inside of Mainnet beta
+
+We have a series of posts in the making that explain the CFD protocol details.
+In this blogpost we give an overview of what is happens under the hood.
+
+In a nutshell there are three players involved when opening and closing a CFD on ItchySats:
+
+1. **Maker**: Publishes offer that defines the CFD (price offered, min value, max value, possible leverage).
+2. **Taker**: Connects to the maker to get an offer and opens positions by taking an offer.
+3. **Oracle**: Periodically signs "prices" (in our case the BTC-USD price). The signed price is used to unlock the payout transaction of the CFD to distribute the funds.
+
+### Opening a position
+
+Let's have a closer look on the communication between the three players when opening a position.
+
+<img alt="OpenPosition" src={useBaseUrl('blog/assets/images/2021-12/OpenPosition.svg')} />
+
+In order to open a position the maker communicates with the oracle to define the period for the CFD.
+The maker picks an oracle event in the future that will sign a price, e.g. the event that will sign the BTC/USD price in 24 hours.
+The oralce event will mark when the CFD period ends.
+The maker then creates an offer.
+
+When a taker connects to a maker the taker will see the offers of the maker.
+The offer includes the maker's price, minumum and maximum quantity, possible leverage and the oracle event to be used.
+The taker can then decide to request to open a position with the maker, taking the offer.
+
+Unless the order is outdated, the maker will accept it. 
+What follows is a communication phase where the parties set up the CFD transactions.
+This is the most complex part, because all the transactions that can later be used for payout are agreed upon by the parties up front.
```

Comment:
```suggestion
This is the most complex part, because all the transactions that can later be used for payouts are agreed upon by the parties up front.
```

Reply:
I think payout is correct here.